load_game.py

```diff
@@ -20,8 +23,97 @@
 FFXC = xbox.controller_handle()
 game_vars = vars.vars_handle()
 
-
-def load_into_game(gamestate: str, step_counter: str):
+def move_after_load(spec_move:str):
+    if spec_move == "miihen_laugh":
+        load_miihen_start_laugh()
+    elif spec_move == "miihen_no_laugh":
+        load_miihen_start()
+    elif spec_move == "MRR":
+        load_mrr()
+    
+
+def load_into_game(gamestate:str, step_counter:str):
+    logger.debug(f"Loading game state {gamestate} | {step_counter}")
+    # If wrong maps are loaded in, try to reset.
+    if memory.main.get_map() not in [23, 348, 349]:
+        reset.reset_to_main_menu()
+    area.dream_zan.new_game(gamestate=gamestate)
+    
+    # Now to get details for the load/save files
+    filepath = "json_ai_files\\save_load_details.json"
```

Comment:
We should be doing:

```python
import os
...
filepath = os.path.join('json_ai_files', 'save_load_details.json')
```

Reply:
Checking a few places we can do that. Stand by.

Reply:
This is updated now.